### PR TITLE
Add audio feature concat.

### PR DESCRIPTION
**Pull request by imamuratakehiro**

### CODE REVIEW

File: lighthouse/common/qd_detr.py [hunk 1]
```diff
@@ -151,6 +151,9 @@ def forward(self, src_txt, src_txt_mask, src_vid, src_vid_mask, src_aud=None, sr
                - "aux_outputs": Optional, only returned when auxilary losses are activated. It is a list of
                                 dictionnaries containing the two above keys for each decoder layer.
         """
+        if src_aud is not None:
+            src_vid = torch.cat([src_vid, src_aud], dim=2)
+
```
[the remark this answers is not in the document]
Comment (imamuratakehiro) [reply lighthouse/common/qd_detr.py:156]: I will check it. Please hold on.

[the remark this answers is not in the document]
Comment (imamuratakehiro) [reply lighthouse/common/qd_detr.py:156]: I modified the codes and confirmed that it works appropriately w/o audio model.

**Comment by awkrail** [comment]
@imamuratakehiro LGTM. Based on your code, I will change the code to apply audio features to other models. After conducting experiments, I will merge the code into main.

File: configs/qvhighlight/clip_audio/qd_detr.yml
Comment (awkrail) [inline configs/qvhighlight/clip_audio/qd_detr.yml:1]: Sorry, I missed it. Could you change the filename from configs/qvhighlight/clip_audio/qd_detr.yml into configs/qvhighlight/clip_pann_qd_detr_qvhighlight.yml (if you use pann for audio features)?

File: configs/qvhighlight/clip_slowfast_audio/qd_detr.yml
Comment (awkrail) [inline configs/qvhighlight/clip_slowfast_audio/qd_detr.yml:1]: Could you change the filename from configs/qvhighlight/clip_slowfast_audio/qd_detr.yml into configs/qvhighlight/clip_slowfast_pann_qd_detr_qvhighlight.yml (if you use pann for audio features)?

File: configs/qvhighlight/clip_audio/qd_detr.yml
Comment (imamuratakehiro) [reply configs/qvhighlight/clip_audio/qd_detr.yml:1]: Understood. I appreciate your feedback.

**Comment by awkrail** [comment]
LGTM. I merge it (then, I will train other models).

**Pull request merged**